aikau/src/main/resources/alfresco/util/hashUtils.js

```diff
@@ -34,12 +34,15 @@ define(["dojo/_base/array",
    var util = {
 
       // See API below
-      getHash: function alfresco_util_hashUtils__getHash() {
+      getHash: function alfresco_util_hashUtils__getHash(suppressDecoding) {
          var hashString = this.getHashString(),
             hashObj = ioQuery.queryToObject(hashString);
-         array.forEach(Object.keys(hashObj), function(hashKey) {
-            hashObj[hashKey] = decodeURIComponent(hashObj[hashKey]);
-         });
+         if (suppressDecoding !== true)
```

Comment:
Really don't like this. It's totally subverting JavaScript's ability to use truthy/falsy values, and it's counter-intuitive. If I wanted to call something like `getHash(myOptions.length)` then it would only work if I prefixed my passed-through value with `!!`. I know it's used a lot elsewhere, so if you want to keep it then I will merge it as-is.